CONTRIBUTING.md

```diff
@@ -131,7 +133,12 @@ __4. Keep a clear git history in mind__
 
 ## How to merge a pull request
 
-Pull request are merged by the maintainer that approved after review. An pull request author should never merge themself, even if they are maintainer. 
+A pull request author should never merge themsel. Pull requests are reviewed by one or more maintainer(s). Once all comments are addressed, the reviewer(s) approve the pull request, and merge it to the `main` branch.
```

Comment:
```suggestion
A pull request author should never merge themselves. 
```

I would be more detailed for "Once all comments are addressed". It's passive, so it's not 100% clear who is addressing comments etc.  I would add something like (of course in better words):

- we will review and leave comments that sometimes are required changes (and we'll explain why it's required), they may be clarifying questions or remarks on possible improvements 
- We'll wait for the person contributing to answer the comments, ask questions, or implement changes until we review again
- once all comments are cleared and everything looks good, we will merge

So basically, I would like to transport the message that we'll take time (means: we appreciate contributions), we'll give feedback or ask questions and if we demand changes explain why (as in: We collaborate on this, we're not only like: NO do this thing I tell you). We will merge when everyone is happy with the change. 

I hope that makes it less scary and a bit more inviting to contribute for some people.

Reply:
Yes, that is a good point!

Reply:
I added more details here: [26c675f](https://github.com/aiven/klaw/pull/1507/commits/26c675f5cf21caea8dc1141c03563e51f865a8a3)

Reply:
I love it 🥹 It's very inviting and clear!